packages/env/lib/wordpress.js

```diff
@@ -68,7 +79,7 @@ async function configureWordPress( environment, config, spinner ) {
 		// Add quotes around string values to work with multi-word strings better.
 		value = typeof value === 'string' ? `"${ value }"` : value;
 		setupCommands.push(
-			`wp config set ${ key } ${ value } --anchor="define( 'WP_DEBUG',"${
+			`wp config set ${ key } ${ value } --anchor=${ configAnchor }${
```

Comment:
Why even use a custom anchor? The default anchor is the line above `/* That's all, stop editing!`, which works across every WordPress version.

Reply:
Not opposed, but this was here before - not sure why, we should probably not change it unless we know it's safe to do so.

Reply:
Looks like this was added in e6d4483d336ffa35027d50c1cafc43ff84782026 / #24884

Some teams used to translate `wp-config-sample.php` for those old versions, but not anymore.

So unless we could use `--anchor=EOF` I suppose we  we should indeed just leave this 👍

Reply:
I'm not sure why it was chosen the way it is, so I'm not sure it's safe to change.

To be clear, I'm not against it, but on the side of caution I'd prefer to keep using the approach that's used so far.